src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -58,9 +58,18 @@ using namespace chip::app::Clusters::WindowCovering;
 
 #define WC_PERCENT100THS_MAX 10000
 
-static bool HasFeature(chip::EndpointId endpoint, Features feature)
+static bool HasFeature(chip::EndpointId endpoint, WindowCoveringFeature feature)
 {
-    return true;
+    uint32_t FeatureMap = 0;
+    if (EMBER_ZCL_STATUS_SUCCESS ==
+        emberAfReadServerAttribute(endpoint, chip::app::Clusters::WindowCovering::Id,
```

Comment:
@eve-cxrp For future reference, please do not use emberAfReadServerAttribute.  Use the `Accessors` functions, that make sure you're using the right type and whatnot, instead of easy-to-mess-up `reinterpret_cast`.